From 6a4bf0ac7e74ea5e40e51024110e03d55c971bfe Mon Sep 17 00:00:00 2001
From: "F. von Gellhorn" <flinux@vongellhorn.ch>
Date: Thu, 10 Dec 2020 23:39:44 +0100
Subject: [PATCH 1/1] Solus tweaks to the default config

- 50000 line scrollback
- show NewTabButton by default
- set false the Highlight new lines
---
 src/profile/Profile.cpp   | 4 ++--
 src/settings/konsole.kcfg | 2 +-
 2 files changed, 3 insertions(+), 3 deletions(-)

diff --git a/src/profile/Profile.cpp b/src/profile/Profile.cpp
index d09d026..299a670 100644
--- a/src/profile/Profile.cpp
+++ b/src/profile/Profile.cpp
@@ -193,10 +193,10 @@ void Profile::useFallback()
     setProperty(Font, QFontDatabase::systemFont(QFontDatabase::FixedFont));
 
     setProperty(HistoryMode, Enum::FixedSizeHistory);
-    setProperty(HistorySize, 1000);
+    setProperty(HistorySize, 50000);
     setProperty(ScrollBarPosition, Enum::ScrollBarRight);
     setProperty(ScrollFullPage, false);
-    setProperty(HighlightScrolledLines, true);
+    setProperty(HighlightScrolledLines, false);
 
     setProperty(FlowControlEnabled, true);
     setProperty(UrlHintsModifiers, 0);
diff --git a/src/settings/konsole.kcfg b/src/settings/konsole.kcfg
index 49b6751..a0dbcba 100644
--- a/src/settings/konsole.kcfg
+++ b/src/settings/konsole.kcfg
@@ -143,7 +143,7 @@
     </entry>
     <entry name="NewTabButton" type="Bool">
       <label>Control the visibility of 'New Tab' button on the tab bar</label>
-      <default>false</default>
+      <default>true</default>
     </entry>
     <entry name="CloseTabButton" type="Enum">
       <label>Control where the "Close tab" button will be displayed</label>
-- 
2.29.2

